Podfile

```diff
@@ -1,9 +1,10 @@
-platform :ios, '8.0'
+platform :ios, '9.0'
```

Comment:
We’ve been trying to keep this at the same minimum deployment target as the SDK, as one more check that we’re not breaking promised backwards compatibility. I’d like to keep it at 8, unless there’s absolutely no way not to.